tests/console/yast2_samba.pm

```diff
@@ -0,0 +1,279 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "consoletest";
+use testapi;
+
+
+
+sub run() {
+    select_console 'root-console';
+
+    # check network at first
+    assert_script_run("if ! systemctl -q is-active network; then systemctl -q start network; fi");
+    # install local ldap server
+    assert_script_run("/usr/bin/zypper -n -q in yast2-auth-server yast2-ldap openldap2 openldap2-client krb5-server krb5-client tdb-tools");
```

Comment:
openSUSE and SLES have different situations regarding to default installed packages. To avoid issues on openSUSE I need to do this step and won't YaST install packages afterwards. This is why Max complained for my previous test.